webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdatePlan.scala

```diff
@@ -47,12 +47,6 @@ object RepositoryUpdatePlan {
       PluginForKnoraBaseVersion(versionNumber = 29, plugin = new UpgradePluginPR3110()),
       PluginForKnoraBaseVersion(versionNumber = 30, plugin = new UpgradePluginPR3111()),
       PluginForKnoraBaseVersion(versionNumber = 31, plugin = new UpgradePluginPR3112()),
-      PluginForKnoraBaseVersion(versionNumber = 32, plugin = new MigrateOnlyBuiltInGraphs()),
-      PluginForKnoraBaseVersion(versionNumber = 33, plugin = new MigrateOnlyBuiltInGraphs()),
-      PluginForKnoraBaseVersion(versionNumber = 34, plugin = new MigrateOnlyBuiltInGraphs()),
-      PluginForKnoraBaseVersion(versionNumber = 35, plugin = new MigrateOnlyBuiltInGraphs()),
-      PluginForKnoraBaseVersion(versionNumber = 36, plugin = new MigrateOnlyBuiltInGraphs()),
-      PluginForKnoraBaseVersion(versionNumber = 37, plugin = new MigrateOnlyBuiltInGraphs()),
```

Comment:
I don't quite understand why these `MigrateOnlyBuiltInGraphs` where removed, but the the others above are still presented. This would mean gaps between knora-base version/plugins here, but since the current version is `37` the next plugin will create the gap anyway, because it will bump the knora-base version to `38` and the last version presented here is `31`.  So IMO make sense to either remove or leave all so far created `MigrateOnlyBuiltInGraphs`. Unless I miss something...

Reply:
I removed the needless plugins: https://github.com/dasch-swiss/dsp-api/pull/3216/commits/c883a517faa57dff8ee4f5bc7ccd44c645a0c45b